Realm/Realm.Fody/Common/Analytics.cs

```diff
@@ -118,7 +116,8 @@ private string JsonPayload
                     .Replace("%SYNC_ENABLED%", _isSyncEnabled.ToString())
 
                     // Version of weaver is expected to match that of the library.
-                    .Replace("%REALM_VERSION%", Assembly.GetExecutingAssembly().GetName().Version.ToString())
+                    // TODO: temp - remove that when we're out of beta
+                    .Replace("%REALM_VERSION%", "10.0.0-beta.1") // Assembly.GetExecutingAssembly().GetName().Version.ToString())
```

Comment:
Shouldn't this be 10.2.0-beta.1?

Reply:
Doh 🤦‍♂️ apparently the sun was too much for me yesterday 😅